src/configuration.php

```diff
@@ -895,7 +909,7 @@ public function saveFile(array $fieldList, $path = null, $validate = true)
 
         // if necessary point $path to  myconfig.php
```

Comment:
would change that comment so it tells the reader that it will fallback to the default case